tools/agents/doctor.py

```
@@ -0,0 +1,185 @@
+            findings.append(("cron-unquoted",
+                             "%s: ‏cron בלי מרכאות: %r" % (name, v)))
+
+    # 2. ‏if: שמפנה ל-output שאיש לא כותב — תנאי שהוא תמיד false,
+    #    וה-job מדולג בירוק לנצח. בדיוק הבאג של #266 בצורתו הבאה.
+    written = set(re.findall(r"echo\s+\"?(\w+)=", body))
+    written |= set(re.findall(r"^\s{4,}(\w+):\s*\$\{\{\s*steps\.", body, re.M))
+    for m in re.finditer(r"\boutputs\.(\w+)", body):
+        if m.group(1) not in written:
+            findings.append(("if-orphan-output",
+                             "%s: ‏`outputs.%s` שאיש אינו כותב — התנאי "
+                             "תמיד false וה-job מדולג בירוק"
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)